network-store-iidm-impl/src/test/java/com/powsybl/network/store/iidm/impl/OperatingStatusExtensionTest.java

```diff
@@ -163,4 +166,104 @@ public void testHvdcLineOperatingStatusExtension() {
 
         assertEquals(1, hvdcLine.getExtensions().size());
     }
+
+    @Test
+    public void testRemoveExtension() {
+        Network network = CreateNetworksUtil.createNodeBreakerNetworkWithLine();
+        DummyNetworkListener listener1 = new DummyNetworkListener();
+        DummyNetworkListenerWithExceptions listener2 = new DummyNetworkListenerWithExceptions();
+        network.addListener(listener1);
+        network.addListener(listener2);
```

Comment:
I don't get the point of the listener with exceptions. Exceptions are thrown as expected but you don't test them ? Is it the purpose of checking that the function does not crash ?

Reply:
Yes it's to test that notifyExtensionBeforeRemoval() and notifyExtensionAfterRemoval() methods don't crash when onExtensionBeforeRemoval() and notifyExtensionAfterRemoval() are not implemented. It is also for coverage